src/controller/tests/TestServerCommandDispatch.cpp

```diff
@@ -151,7 +151,7 @@ void TestCommandInteraction::TestNoHandler(nlTestSuite * apSuite, void * apConte
     ctx.EnableAsyncDispatch();
 
     chip::Controller::InvokeCommandRequest(&ctx.GetExchangeManager(), sessionHandle, kTestEndpointId, request, onSuccessCb,
-                                           onFailureCb);
+                                           onFailureCb, Optional<System::Clock::Timeout>());
```

Comment:
```suggestion
                                           onFailureCb, NullOptional);
```
again, assuming we don't just default to that.

Reply:
defaulted.